libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/children-residence-change.service.ts

```diff
@@ -146,15 +146,14 @@ export class ChildrenResidenceChangeService {
     const { answers } = application
     const { counterParty } = answers
 
-    // TODO Remove null check on counter party once we add it to the template.
-    if (counterParty?.email) {
+    if (counterParty.email) {
       await this.sharedTemplateAPIService.sendEmail(
         transferRequestedEmail,
         (application as unknown) as Application,
       )
     }
 
-    if (counterParty?.phoneNumber) {
+    if (counterParty.phoneNumber) {
```

Comment:
Not in this pr, but we should add unit tests for this service. I'll add a ticket for it.